packages/apps/transfer/src/pages/faucet/existing/index.tsx

```diff
@@ -34,56 +38,36 @@ const ExistingAccountFaucetPage: FC = () => {
   const [chainID, onChainSelectChange] = usePersistentChainID();
 
   const [requestStatus, setRequestStatus] = useState<{
-    status: RequestStatus;
+    status: FormStatus;
     message?: string;
-  }>({ status: 'not started' });
-
-  const onPoll = async (
-    transaction: IPactCommand & ICommandBuilder<Record<string, unknown>>,
-    pollRequest: Promise<IPollResponse>,
-  ): Promise<void> => {
-    const request = await pollRequest;
-    const result = request[transaction.requestKey!]?.result;
-    const status = result?.status;
-    if (status === 'failure') {
-      const apiErrorMessage = (result.error as { message: string }).message;
-
-      setRequestStatus({ status: 'failed', message: apiErrorMessage });
-    }
-  };
+  }>({ status: 'idle' });
 
   const onFormSubmit = useCallback(
     async (e: FormEvent) => {
       e.preventDefault();
 
-      setRequestStatus({ status: 'pending' });
+      setRequestStatus({ status: 'processing' });
 
       try {
-        await fundExistingAccount(
-          accountName,
-          chainID,
-          onPoll,
-          AMOUNT_OF_COINS_FUNDED,
-        );
+        await fundExistingAccount(accountName, chainID, AMOUNT_OF_COINS_FUNDED);
 
-        setRequestStatus({ status: 'succeeded' });
+        setRequestStatus({ status: 'successful' });
```

Comment:
Perhaps we could use a string enum to hold the possible statuses.

Reply:
We're already achieving that, `status` is of `type FormStatus = 'idle' | 'successful' | 'erroneous' | 'processing';`